packages/syntax-core/src/RichSelect/RichSelectList.tsx

```diff
@@ -176,31 +178,31 @@ function RichSelectList(props: RichSelectListProps): ReactElement {
       <div
         className={classNames(styles.selectContainer, {
           [styles.opacityOverlay]: disabled,
+          [styles.selectContainerCambio]: themeName === "cambio",
         })}
       >
-        <ReactAriaLabel
```

Comment:
`ReactAriaLabel` renders a `label` and we rendered a `label` inside of it. Fix that